### ISSUE TITLE
Issue with Left OUTER JOIN/RIGHT OUTER JOIN 

### ISSUE THREAD

**Issue by Keshavkumbham** [darold/ora2pg#524]
ORACLE query: 
```
SELECT OC.CASE_ID,OC.EMAIL_QUEUE_NAME
FROM 
(select r.reg_id
from o_org_unit_marketplace_map r where 
r.legal_entity_id=vg.legal_entity_id 
) pl
,(SELECT CASE_ID,REG_ID,EMAIL_QUEUE_NAME
from TAM_DDL.O_CASES
) OC
WHERE PL.VEN_NAME = OC.NAME(+) 
AND PL.reg_id=OC.REG_ID(+);
```
CONVERTED query:
```
SELECT OC.CASE_ID,OC.EMAIL_QUEUE_NAME
FROM  
_**(SELECT CASE_ID,REG_ID,EMAIL_QUEUE_NAME
from TAM_DDL.O_CASES
) oc, (select r.reg_id
from o_org_unit_marketplace_map r where
r.legal_entity_id=vg.legal_entity_id 
) pl,**_ (select r.reg_id
from o_org_unit_marketplace_map r where 
r.legal_entity_id=vg.legal_entity_id 
) pl
LEFT OUTER JOIN (SELECT CASE_ID,REG_ID,EMAIL_QUEUE_NAME
from TAM_DDL.O_CASES
) oc ON (PL.VEN_NAME = OC.NAME AND PL.reg_id = OC.REG_ID);
```
ISSUE: 
In the above converted query, the bold part is unwanted or redundant part of conversion. Please look into it.

**Comment by darold** [comment]
Might be solved with commit 64bd16a

**Comment by Keshavkumbham** [comment]
Thanks darold. It's fixed.
